modules/library/main/src/main/java/org/geotools/filter/function/FilterFunction_mapGet.java

```diff
@@ -0,0 +1,75 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2020, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.filter.function;
+
+import static org.geotools.filter.capability.FunctionNameImpl.parameter;
+
+import java.util.Map;
+import org.geotools.factory.CommonFactoryFinder;
+import org.geotools.filter.FunctionExpressionImpl;
+import org.geotools.filter.capability.FunctionNameImpl;
+import org.geotools.util.Converters;
+import org.opengis.filter.FilterFactory2;
+import org.opengis.filter.capability.FunctionName;
+import org.opengis.filter.expression.PropertyName;
+import org.opengis.filter.expression.VolatileFunction;
+
+/** Extracts a value from a map given the mapname.key identifier as a parameter */
+public class FilterFunction_mapGet extends FunctionExpressionImpl implements VolatileFunction {
```

Comment:
@jodygarnett is not happy with these names :-D

```suggestion
public class MapGetFunction extends FunctionExpressionImpl implements VolatileFunction {
```

Reply:
ok. Thanks for the hint.
I will rename it. (I have seen a ton of FilterFunction_something so I used that :) Probably old naming convention.)